workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -23,6 +23,7 @@ type Query {
   endpointChanges(sinceBatchCommitId: String): EndpointChanges
   batchCommits: [BatchCommit]
   diff(diffId: ID): DiffState
+  hash: String
```

Comment:
@niclim was thinking of exposing a generic metadata projection, I think this might fit better aligned with that

Reply:
Yep, that makes sense. `hash` feels a bit naked here as it is, it could certainly use some company.